Unified.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
```

Comment:
"Unified.py" is actually not a good file name. Think of the reader perspective, what is unified? The reader does not know you unified two scripts, so the name does not make sense to him, nor give any information about what this file does.